src/resources/2022/GreyGoose.ts

```diff
@@ -0,0 +1,53 @@
+import { have as have_ } from "../../lib";
+import { get as getModifier } from "../../modifier";
+import { get } from "../../property";
+import { $familiar, $item, $skill } from "../../template-string";
+
+export const goose = $familiar`Grey Goose`;
+
+export function have(): boolean {
+  return have_(goose);
+}
+
+export function currentExperience(): number {
```

Comment:
does the familiar data type's "experience" function default to missing, or zero? according to the wiki for [proxy records](https://wiki.kolmafia.us/index.php/Proxy_Records), it defaults to 0, which to me would indicate this wouldn't actually work. since a zero XP goose doesn't *always* have 81-100 XP incoming, it is possible the user used it up in a fight and didn't conclude the fight naturally (a la saber, freerun, etc). i could be wrong tho! was this tested?

Reply:
It does, and I just accounted for that by checking the only thing that can make it roll back down to 0 exp